docs/hazmat/primitives/cryptographic-hashes.rst

```diff
@@ -117,6 +117,35 @@ SHA-2 family
     SHA-512 is a cryptographic hash function from the SHA-2 family and is
     standardized by NIST. It produces a 512-bit message digest.
 
+BLAKE2
+~~~~~~
+
+`BLAKE2`_ is a cryptographic hash function specified in :rfc:`7693`. It
+supports keying, personalization, and salting.
+
+.. note::
+
+    As of OpenSSL 1.1.0 BLAKE2 is only supported as an unkeyed hash with no
+    personalization or salting features.
```

Comment:
I think it's confusing to mention all these features that we don't have, maybe it's better to just omit them for now? Or at least drop them from the opening paragraph and make the note "While the RFC describes <features> for Blake2, currently these are not supported."